### PR TITLE
Refs #12970 - Update bootstrap-sass to 3.3.6 for Patternfly

### PR DESCRIPTION
**Pull request by dLobatog**
https://github.com/theforeman/foreman-packaging/pull/944#issue-124499878

Edit (DC): depends on #950, runtime

### CODE REVIEW

File: rubygem-bootstrap-sass/rubygem-bootstrap-sass.spec [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 Summary: Twitter's Bootstrap, converted to Sass and ready to drop into Rails or Compass
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.0.3.0
-Release: 3%{?dist}
+Version: 3.3.6
+Release: 1%{?dist}
 Group: Development/Languages
 License: ASL 2.0
```
Comment (domcleal) [inline rubygem-bootstrap-sass/rubygem-bootstrap-sass.spec]: Changed to MIT

File: rubygem-bootstrap-sass/rubygem-bootstrap-sass.spec [hunk 2]
```diff
@@ -5,8 +5,8 @@
 
 Summary: Twitter's Bootstrap, converted to Sass and ready to drop into Rails or Compass
```
Comment (domcleal) [inline rubygem-bootstrap-sass/rubygem-bootstrap-sass.spec]: This changed to `bootstrap-sass is a Sass-powered version of Bootstrap 3, ready to drop right into your Sass powered applications.` (though it'll need trimming to 72 chars).

**Comment by domcleal** [comment]
Note to self, this can't be built until the corresponding Foreman change is made as there's a hard dep from Foreman on the existing version.

**Comment by dLobatog** [comment]
@domcleal Updated according to your comments, thanks! I updated the foreman spec as well now that it's merged

File: foreman/foreman.spec [hunk 3]
```diff
@@ -190,8 +190,9 @@ BuildRequires: %{?scl_prefix_ror}rubygem(execjs) >= 1.4.0
 BuildRequires: %{?scl_prefix_ror}rubygem(execjs) < 2.5.0
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) >= 3.1
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) < 4.0
+BuildRequires: %{?scl_prefix_ror}rubygem(bootstrap-sass) >= 3.3.5
```
Comment (domcleal) [inline foreman/foreman.spec]: This shouldn't be in foreman.spec should it?  It looks like there's no dep in Foreman, only in patternfly-sass.

**Comment by domcleal** [comment]
Could you rebase this please @dLobatog?  (This may become a frequent request, sorry.)

**Comment by dLobatog** [comment]
Rebased

**Comment by domcleal** [comment]
Merged and building, thanks @dLobatog.

**Pull request merged**